spec/controllers/saml_idp_controller_spec.rb

```diff
@@ -1638,7 +1644,7 @@ def name_id_version(format_urn)
         )
       end
 
-      shared_examples_for 'sends the UUID' do
+      shared_examples_for 'sends the UUID' do |requested_nameid_format|
```

Comment:
we could probably default this to `nil` so the specs don't have to pass in a blank format for specs that don't expect it?
```suggestion
      shared_examples_for 'sends the UUID' do |requested_nameid_format=nil|
```

Reply:
Normally that would make sense. But here, `nil` is not a default or don't care. It is explicitly the test case for when the SAML request does not explicitly request a NameID format. See lines 1732 and 1736.